src/views/Main.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from 'react';
+import getCountries from '../services/Country';
+import CountryCard from '../components/CountryCard/CountryCard';
+import Dropdown from '../components/Dropdown/Dropdown';
+import './Main.css';
+
+export default function Main() {
+  const [countries, setCountries] = useState ([]);
+  const continent = ['Asia', 'Africa', 'North America', 'South America', 'Antarctica', 'Europe', 'Oceania'];
+  const [cont, setCont] = useState ('Asia');
+  const [load, setLoad] = useState(true);
+
+  useEffect(()=>{
+    const fetchCountries = async () => {
+
+     
+
+      try {
+        const response = await getCountries();
+        setCountries(response);
+      } catch (e) {
+        alert(e.message);
+      }
+      setTimeout(() => {
+        setLoad(false);
+      }, 1000);
```

Comment:
To get the data to load after each item comes back from the database, we can usually place setLoad(false) under where we call setCountries(response) within our try block, no setTimeout would be needed in this instance. This will allow all the countries to be set in state and then the loading spinner will stop.